src/scripts/components/NotificationHub.js

```diff
@@ -57,21 +63,23 @@ export const NotificationHub = ( { initialActive = false } ) => {
 	}, [ isActive ] );
 
 	return (
-		<ShortcutProvider
-			className={ classNames( [
-				'notifications',
-				isActive ? 'active' : '',
-			] ) }
-		>
-			<NotificationHubIcon
-				toggle={ toggleDrawer }
-				isActive={ isActive }
-			/>
-			<Drawer
-				instance={ drawerRef }
-				focus={ () => setIsActive( true ) }
-				blur={ () => setIsActive( false ) }
-			/>
-		</ShortcutProvider>
+		<div className="ab-item ab-empty-item" tabIndex={ 0 }>
+			<ShortcutProvider
+				className={ classNames( [
+					'notifications',
+					isActive ? 'active' : '',
+				] ) }
+			>
+				<NotificationHubIcon
+					toggle={ toggleDrawer }
+					isActive={ isActive }
+				/>
+				<Drawer
+					instance={ drawerRef }
+					focus={ () => setIsActive( true ) }
+					blur={ () => setIsActive( false ) }
+				/>
+			</ShortcutProvider>
+		</div>
```

Comment:
Wrapping this in `div.ab-item.ab-empty-item`  creates the same HTML structure as WordPress creates for the menu item.